src/pages/tiptap.tsx

```diff
@@ -0,0 +1,181 @@
+import { useEditor, EditorContent } from '@tiptap/react'
+import StarterKit from '@tiptap/starter-kit'
+import CharacterCount from '@tiptap/extension-character-count'
+import { NextPage } from 'next'
+import { useState } from 'react'
+import { Navbar } from '../components/Navbar'
```

Comment:
I would get rid of the double Navbar. Put the props for the profile page on the right of the editor bar you have.